redfish-core/lib/power_supply.hpp

```diff
@@ -205,12 +205,14 @@ inline void getValidPowerSupplyPath(
 
 inline void
     getPowerSupplyState(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
-                        const std::string& service, const std::string& path)
+                        const std::string& service, const std::string& path,
+                        bool available)
 {
     sdbusplus::asio::getProperty<bool>(
         *crow::connections::systemBus, service, path,
         "xyz.openbmc_project.Inventory.Item", "Present",
-        [asyncResp](const boost::system::error_code& ec, const bool value) {
+        [asyncResp, available](const boost::system::error_code ec,
```

Comment:
hmm, I guess not resolved yet even though I thought a pushed the fix.  will investigate.